src/json/motion.rs

```diff
@@ -96,11 +248,12 @@ fn json_samples_motion3() {
                 continue;
             }
 
-            serde_json::from_str::<Motion3>(
+            let motion = serde_json::from_str::<Motion3>(
                 &std::fs::read_to_string(&motion)
                     .expect(&format!("error while reading: {:?}", motion)),
             )
             .expect(&format!("error while parsing: {:?}", motion));
+            println!("{:#?}", motion);
```

Comment:
This print statement shouldnt be part of the test, I assume you used this for debugging and forgot to remove it?

Reply:
Oops, I just forgot to remove this... Also fixed in 177e920.